src/components/ForgotPassword.tsx

```diff
@@ -0,0 +1,356 @@
+"use client";
+
+import React, { useState, useRef } from "react";
+import Image from "next/image";
+import TextField from "@mui/material/TextField";
+import { Icon } from "@iconify/react/dist/iconify.js";
+import { useRouter } from "next/navigation";
+
+import InputAdornment from "@mui/material/InputAdornment";
+import IconButton from "@mui/material/IconButton";
+import VisibilityOffIcon from "@mui/icons-material/VisibilityOff";
+import VisibilityIcon from "@mui/icons-material/Visibility";
+
+export default function ForgotPasswordForm() {
+  const router = useRouter();
+
+  const [step, setStep] = useState(1);
+  const [email, setEmail] = useState("");
+  const [showNewPassword, setShowNewPassword] = useState(false);
+  const [showConfirmPassword, setShowConfirmPassword] = useState(false);
+  const [newPassword, setNewPassword] = useState("");
+  const [confirmPassword, setConfirmPassword] = useState("");
+  const [code, setCode] = useState(["", "", "", ""]);
+  const [codeError, setCodeError] = useState(false);
+  const [codeBackendError, setCodeBackendError] = useState(false);
+  const inputs = useRef<(HTMLInputElement | null)[]>([]);
+  const [error, setError] = useState("");
+  const [passwordError, setPasswordError] = useState(false);
+  const [counter, setCounter] = React.useState(30);
+
+  const handleEmailSubmit = () => {
```

Comment:
Try to make it so if you press enter in the text field it allows you to continue